### PR TITLE
Added XP-Pen Deco mini 4 config

### PR DESCRIPTION
**Pull request by Ivanoks**

### CODE REVIEW

**Review by InfinityGhost — CHANGES_REQUESTED** [review]
TABLETS.md must be updated to include this tablet. It should include any of its missing features, quirks, etc. in order to be accepted.

**Comment by Ivanoks** [comment]
I asked a friend who has a Deco mini 4, he said that everything works.

**Comment by Mrcubix** [comment]
You need to add a new entry in `Tablets.md` with the tablet, the status (which will be "Missing Features if the wheel doesn't work (if it has one)) and the note (if it's missing features, which ones).

Preferably add it before `XP-Pen Deco Mini7`.

**Comment by Ivanoks** [comment]
I apologize very much for the delay, the school just caught up with me, in short, I updated the config, and it works too. (I fucked up with the push level.)

File: OpenTabletDriver/Configurations/XP-Pen/Deco mini 4.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "DigitizerIdentifiers": [
+    {
+      "VendorID": 10429,
+      "ProductID": 2345,
+      "InputReportLength": 12,
+      "OutputReportLength": 12,
+      "ReportParser": "OpenTabletDriver.Vendors.XP_Pen.XP_PenReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": [
+        "ArAEAAAAAAAAAAAA"
```
Comment (X9VoiD) [inline OpenTabletDriver/Configurations/XP-Pen/Deco mini 4.json:31]: No need to specify trailing zeroes.

```suggestion
        "ArAE"

```

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver/Configurations/XP-Pen/Deco mini 4.json (hunk 1 above)
Comment (Ivanoks) [reply OpenTabletDriver/Configurations/XP-Pen/Deco mini 4.json:31]: Oh okay, sorry.

**Review by X9VoiD — APPROVED** [review]
Ah, I mistakenly added a newline, other than that the config is fine now. Btw, what is the "missing feature"?

**Comment by Ivanoks** [comment]
> Ah, I mistakenly added a newline, other than that the config is fine now. Btw, what is the "missing feature"?

Nothing, Deco mini7 has nothing written, although it is in missing feature.

**Comment by X9VoiD** [comment]
Mark it is `Supported`.

**Comment by Ivanoks** [comment]
> Mark it is `Supported`.

Did.

**Pull request merged**